lib/cargo.js

```diff
@@ -13,10 +22,12 @@ function provideBuilder() {
     },
 
     settings: function (path) {
+      var cargoPath = atom.config.get('build-cargo.cargoPath');
```

Comment:
Is the default setting is not taken from the description of the item settings?
line 8